packages/core/src/sass/app/modules/notifications-dialog.scss

```diff
@@ -47,7 +47,7 @@
     &-dialog {
         position: absolute;
         top: 48px;
-        left: calc(-50% - 10rem);
+        left: calc(-50% - 18rem);
```

Comment:
@heorhi-deriv wouldn't this cause the notification list to be shifted for pages which do not need the shift (i.e cashier pages)
Before:
<img width="1292" alt="image" src="https://github.com/user-attachments/assets/b8bfe7fb-767e-448d-a982-157eb5d0dfa5">
After:
<img width="1292" alt="image" src="https://github.com/user-attachments/assets/56092068-42c0-41de-9c81-e2e2f8664b54">

Maybe a better option is to add the pages which needs shifting (i.e the wallets and compare accounts routes) to the existing pre-appstore variant instead

Reply:
mmm 🤔
nice catch let me check

Reply:
hey hey @lubega-deriv 
This is sorted.
Instead of checking the routes i decided to check that notification tray is outside the viewport, and if so we are adding a class which is shifting the notification tray.